integration/templates/tanstack-start/app/routes/__root.tsx

```diff
@@ -3,9 +3,9 @@ import {
   ScrollRestoration,
   createRootRoute,
   Scripts,
+  HeadContent,
```

Comment:
https://github.com/TanStack/router/discussions/2863#discussioncomment-12206026

Reply:
I did a quick follow up [commit](https://github.com/clerk/javascript/pull/5234/commits/057907760de9c21ae9e94b2d20b47bbdb525370b) to https://github.com/clerk/javascript/pull/5234 which has an incorrect import